lib/matplotlib/animation.py

```diff
@@ -446,7 +446,7 @@ class FFMpegFileWriter(FileMovieWriter, FFMpegBase):
     def _args(self):
         # Returns the command line parameters for subprocess to use
         # ffmpeg to create a movie using a collection of temp images
-        return [self.bin_path(), '-i', self._base_temp_name(),
+        return [self.bin_path(), '-r', str(self.fps), '-i', self._base_temp_name(),
                 '-vframes', str(self._frame_counter),
                 '-r', str(self.fps)] + self.output_args
```

Comment:
If you closely read the link in the original post, you'll see that the first `-r` controls the rate for the *raw* input, while the second `-r` controls the rate for the output. Blame the ffmpeg guys for their command-line syntax.